gp-unique-id/gpuid-wait-for-payment-support-for-payfast.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Gravity Perks // GP Unique ID // Add Wait-for-payment Support for PayFast
+ * https://gravitywiz.com/documentation/gp-unique-id/
+ *  
+ * This snippet adds support to capture payment made via PayFast before generating the Unique ID.
```

Comment:
As a general rule, let's ditch the "This snippet..." intro and just state what the code does. For example, this would become:

"Capture a payment made via PayFast before generating the Unique ID."